### PR TITLE
Update shubhsharma19.json

### PR DESCRIPTION
**Pull request by shubhsharma19**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
:) have a great weekend

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4153"></a>

### CODE REVIEW

**Comment by shubhsharma19** [comment]
Yipee Fixed the conflict issue myself :)

**Review by rahulpandey70 — DISMISSED** [review]
LGTM

File: data/shubhsharma19.json [hunk 1]
```diff
@@ -3,47 +3,63 @@
...
+    "UIUX",
+    "Content Creation"
   ],
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/code_shubh" },
     { "icon": "FaGithub", "url": "https://github.com/shubhsharma19" },
     { "icon": "SiHashnode", "url": "https://shubhsharma19.hashnode.dev/" },
-    { "icon": "FaInstagram", "url": "https://instagram.com/code.shubh/" }
+    { "icon": "FaInstagram", "url": "https://instagram.com/code.shubh/" },
+    { "icon": "FaInstagram", "url": "https://instagram.com/code.shubh/" },
```
Comment (kumarsonsoff3) [inline data/shubhsharma19.json:23]: Seems like you've added same Instagram link twice. Please remove one of them. 

```suggestion
```

File: data/shubhsharma19.json [hunk 2]
```diff
@@ -3,47 +3,63 @@
   "type": "personal",
   "displayStatsPublic": true,
   "bio": "BCA Graduate | **Open Source** Advocate | Fullstack Web Development | Designer and Content Creator | Tech Blogger | Music Producer",
+  "bio": "BCA Graduate | **Open Source** Advocate | Fullstack Web Development | Designer and Content Creator | Technical Writer | Music Producer",
```
Comment (kumarsonsoff3) [inline data/shubhsharma19.json:6]: You've added same bio twice. Please remove one of them. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please remove all the duplicate fields from the file, like you've used the `name` twice for your links , `description`, `title` keys multiple times for same milestone.

Let us know if you need any help with this!

**Comment by shubhsharma19** [comment]
Fixed it

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Looks like there are duplicated fields in the links. You might want to remove one of those.

**Comment by shubhsharma19** [comment]
Fixed :)

**Pull request merged**